changes/test_tasks.py

```diff
@@ -80,6 +82,35 @@ def create_outbound_lookup(self, result_count=1):
             match_querystring=True,
         )
 
+    def create_identity_lookup_with_id_field(self, lang="eng_ZA"):
+        responses.add(
+            responses.GET,
+            "http://is/api/v1/identities/test-identity-uuid/",
+            json={
+                "results": [
+                    {"id": "test_identity-uuid", "details": {"lang_code": lang}}
+                ]
+            },
+            status=200,
+            match_querystring=True,
+        )
+
+    def update_identity_lookup_with_id_field(self, lang="eng_ZA"):
+        responses.add(
+            responses.PATCH,
+            "http://is/api/v1/identities/test-identity-uuid/",
+            json={
+                "results": [
+                    {
+                        "id": "test_identity-uuid",
+                        "details": {"lang_code": lang, "timeout_timestamp": "time"},
+                    }
+                ]
```

Comment:
The `PATCH` action on this endpoint will just return the updated identity, not a list of identities.